Sources/MessagingInApp/Gist/Gist.swift

```diff
@@ -156,12 +159,6 @@ public class Gist: GistInstance, GistDelegate {
     func removeMessageManager(instanceId: String) {
         messageManagers.removeAll(where: { $0.currentMessage.instanceId == instanceId })
     }
-
-    // Inline messages
-
-    func getInlineMessages(forElementId elementId: String) -> [Message] {
-        messageQueueManager.getInlineMessages(forElementId: elementId)
-    }
 }
```

Comment:
I decided to call `getInlineMessages` directly from `MessageQueueManager` instead of `Gist` to reduce amount of boilerplate code needed.